app/views/govuk_publishing_components/components/_inverse_header.html.erb

```diff
@@ -1,10 +1,12 @@
 <%
   full_width ||= false
   header_class = full_width ? "gem-c-inverse-header--full-width" : ""
+  no_padding_top ||= false
+  padding_top_class = no_padding_top ? "gem-c-inverse-header--no-padding-top" : ""
```

Comment:
An alternative to these ternary conditions would be to have an array of css classes...
```ruby
css_classes = []
css_classes << "gem-c-inverse-header--full-width" if full_width
css_classes << "gem-c-inverse-header--no-padding-top" if no_padding_top
```
then used like...
```ruby
<header class="gem-c-inverse-header<%= css_classes.join(' ') %>">
```